ocrd_monitor/flask/app/__init__.py

```diff
@@ -2,25 +2,16 @@
 
 from flask import Flask
 
-def create_app(test_config=None):
+def create_app():
     # create and configure the app
     app = Flask(__name__, instance_relative_config=True)
+
     app.config.from_mapping(
-        SECRET_KEY='dev',
-        DATABASE=os.path.join(app.instance_path, 'flaskr.sqlite'),
-        BASEDIR=".",
-        BWHOST="localhost",
-        BWPORT=8085,
-        LOGPORT=8088,
+        BASEDIR=os.getenv("APP_BASEDIR"),
+        BW_PORT=os.getenv("APP_BW_PORT"),
+        LOG_PORT=os.getenv("APP_LOG_PORT"),
     )
 
-    if test_config is None:

```

Comment:
No more `config.py` mechanism?

Reply:
Atm we do not have such a config and there exist different ways to do a configuration without extra config.py e.g. .env file for flask app.

Reply:
Ok!